### PR TITLE
added Stringbuilder

### PR DESCRIPTION
**Pull request by taron19**

### CODE REVIEW

**Comment by taron19** [comment]
i added stringbuilder

File: Hel.java [hunk 1]
```diff
@@ -1,6 +1,8 @@
 public class Hel {
     public static void main(String[] args) {
         System.out.println("new class");
+        StringBuilder stringBuilder = new StringBuilder("hello");
```
Comment (taron1991) [inline Hel.java]: hey change this to buffer

Comment (taron19) [reply Hel.java]: ok man

File: Hel.java [hunk 2]
```diff
@@ -1,6 +1,8 @@
 public class Hel {
     public static void main(String[] args) {
         System.out.println("new class");
+        StringBuffer stringBuilder = new StringBuffer("hello");
+        stringBuilder.append("ss").append("a");
```
Comment (taron1991) [inline Hel.java:5]: this is right

**Comment by taron19** [comment]
my man what i gotta do

**Pull request merged**